cosim/tcl/vivado-utils.tcl

```diff
@@ -83,6 +83,7 @@ proc vivado_elab_wrap { do_elab proj_bd } {
 
 proc vivado_synth_wrap { do_synth threads } {
     if {${do_synth}} {
+        #set_property strategy Flow_RuntimeOptimized [get_runs synth_1]
```

Comment:
if leaving this commented out, perhaps add a comment above as to why (e.g., "uncomment to use a different synth strategy"